stylesheets/components/button.css

```diff
@@ -149,13 +151,16 @@
 
 .button--icon svg,
 .button--icon .button__icon {
-    height: 1em;
     fill: currentColor;
     color: currentColor;
     left: 0;
     margin: 0;
 }
 
+.button--icon .button__icon {
```

Comment:
This selector is duplicated. Intentional?

Reply:
line 152-153 apply rules that are the same for 2 selectors `.button--icon svg` and `.button--icon .button__icon` 
line 160 `.button--icon .button__icon` adds 1 additional rule for this selector only

The `.button--icon` thing maybe worth explaining: 
We have regular buttons with text (`.button` > some text), regular buttons with icon and text (`.button` > `i.fa.fa-icon.button_icon`, some text), and "icon only" square buttons (`.button--icon` > some icon (svg|font-awesome|other), no text)

The `.button--icon .button__icon` rule is for safety, as we don't expect any `.button__icon` class inside of `.button--icon` but in case it's there, its styles should be reset (that's what line 153 does)

Reply:
Ah, my bad. I didn't  look up  as far as 152. 🤦‍♂